src/DivideIQ.php

```diff
@@ -19,6 +19,7 @@
 
 use GuzzleHttp\Client as HttpClient;
 use GuzzleHttp\Exception\RequestException;
+use HttpResponseException;
```

Comment:
Removed it, was using this import for error handling with guzzle, was testing so removed it in commit: https://github.com/DivideBV/PHPDivideIQ/pull/1/commits/0fee665021840c10534f636300edf122cc95c3fb